posthog/client.py

```diff
@@ -330,9 +330,11 @@ def feature_enabled(self, key, distinct_id, default=False):
         if not self.feature_flags:
             response = default
         else:
-            try:
-                feature_flag = [flag for flag in self.feature_flags if flag["key"] == key][0]
-            except IndexError:
+            for flag in self.feature_flags:
+                if flag["key"] == key:
+                    feature_flag = flag
+                    break
+            else:
```

Comment:
Not sure if it's Github display, but this `else` seems indented incorrectly? 👀

Reply:
Actually, this is correct, it's a Python for-else loop: https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loop

Reply:
Else will run if break doesn't

Reply:
Ah, fascinating! Wasn't aware this was possible :o